ui/hooks/useCopyToClipboard.js

```diff
@@ -14,14 +14,23 @@ const DEFAULT_DELAY = MINUTE;
 
 /**
  * @param delay - delay in ms
- * @returns {[boolean, (text: string) => void]}
+ * @param opts - clipboard options
+ * @param opts.expireClipboard - expires clipboard after delay
+ * @returns {[boolean, (text: string) => void, () => void]}
  */
-export function useCopyToClipboard(delay = DEFAULT_DELAY) {
+export function useCopyToClipboard(
+  delay = DEFAULT_DELAY,
+  opts = { expireClipboard: true },
+) {
   const [copied, setCopied] = useState(false);
   const startTimeout = useTimeout(
     () => {
-      copyToClipboard(' ', COPY_OPTIONS);
-      setCopied(false);
+      if (copied === true) {
+        if (opts.expireClipboard) {
+          copyToClipboard(' ', COPY_OPTIONS);
+        }
+        setCopied(false);
+      }
```

Comment:
Tested, the SRP copy flow still uses the expire. Only places that explicitly set this to false will not expire (e.g. the public address copy button)